### ISSUE TITLE
add assessment dataset `ebola_cases.csv` to `episodes/data/` location for reuse

### ISSUE THREAD

**Issue by avallecam** [epiverse-trace/tutorials#89]
This will make the dataset reusable by all episode developers by rebasing all our feature branches.

Additionally, from the `rewrite-R-ecology-lesson`, I found that the rendered website can create a link to download that dataset with the path `epiverse-trace.github.io/tutorials/data/name-of-data-set.bbb`. Please, see the [Download the data section](https://www.michaelc-m.com/Rewrite-R-ecology-lesson/#download-the-data) of the lesson.

In summary, the steps would be:
- create new feature branch
- add dataset to `episodes/data/`
- push content to remote
- request a PR, then merge
- request episode developers to rebase
